test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -1051,24 +1051,28 @@ public void AbortingTheConnectionSendsFIN()
             {
                 using (var connection = server.CreateConnection())
                 {
-                    // Never send the body so CopyToAsync always fails.
+                    // Full request and response
```

Comment:
I had to change this because we don't necessarily drain all queued writes before aborting for a FIN.